client/core/core.go

```diff
@@ -2333,18 +2371,33 @@ func (c *Core) MaxSell(host string, base, quote uint32) (*OrderEstimate, error)
 	if err != nil {
 		return nil, fmt.Errorf("%s wallet MaxOrder error: %v", unbip(base), err)
 	}
-	sellRedemptionPer, err := quoteWallet.RedemptionFees()
+
+	dc, _, err := c.dex(host)
 	if err != nil {
-		return nil, fmt.Errorf("%s RedemptionFees error: %v", unbip(quote), err)
+		return nil, err
 	}
 
-	return &OrderEstimate{
-		Lots:           maxSell.Lots,
-		Value:          maxSell.Value,
-		MaxFees:        maxSell.MaxFees,
-		EstimatedFees:  maxSell.EstimatedFees,
-		Locked:         maxSell.Locked,
-		RedemptionFees: maxSell.Lots * sellRedemptionPer,
+	// Estimate a quote-converted lot size.
+	lotSize := quoteAsset.LotSize
+	book, found := dc.books[marketName(base, quote)]
```

Comment:
The `books` map can change out from under us in `handleReconnect` and a few spots in bookie.go.